model/span.go

```diff
@@ -30,20 +30,26 @@ type SpanModel struct {
 	Shared         bool              `json:"shared"`
 	LocalEndpoint  *Endpoint         `json:"localEndpoint,omitempty"`
 	RemoteEndpoint *Endpoint         `json:"remoteEndpoint,omitempty"`
-	Annotations    []Annotation      `json:"annotations"`
-	Tags           map[string]string `json:"tags"`
+	Annotations    []Annotation      `json:"annotations,omitempty"`
+	Tags           map[string]string `json:"tags,omitempty"`
```

Comment:
We dont' want to send `annotations:null` nor `tags:null` in the payload.

Reply:
Agreed... this has already been handled with in #23